website_logo/views/company_view.xml

```diff
@@ -6,11 +6,11 @@
             <field name="model">res.company</field>
             <field name="inherit_id" ref="base.view_company_form"></field>
             <field name="arch" type="xml">
-                <page string="Configuration" position="inside">
```

Comment:
Whaaaaaat?! That's amazing! Does this carry over to all xml files, or just views?